docs/content/_index.md

```diff
@@ -1,42 +1,70 @@
 ---
-title: "Infection Monkey Documentation Hub"
+title: "Infection Monkey Documentation"
 date: 2020-05-26T18:15:37+03:00
 draft: false
 ---
 
-# Infection Monkey documentation hub
+# Infection Monkey documentation
 
-{{< homepage_shortcuts >}}
+
+![Infection Monkey Documentation Logo](/images/monkey-teacher.svg?height=300px
+"Infection Monkey Documentation Logo")
 
 ## What is Guardicore Infection Monkey?
+Infection Monkey is an open-source adversary emulation platform that helps you
+improve your security posture using empirical data. Monkey uses various methods
```

Comment:
```suggestion
improve your security posture using empirical data. It uses various methods
```

Reply:
How strongly do you feel? I kind of like it the original way.

Reply:
I don't feel that strongly about it, but usually when we refer to Infection Monkey, we say "Infection Monkey" or "The Monkey." Just "Monkey" sounds like we've personified it

Reply:
I'll go with "The Monkey".